addons/website_payment/static/tests/tours/donation.js

```diff
@@ -64,6 +65,91 @@ registerWebsitePreviewTour(
             content: "Verify that other_amount is not checked",
             trigger: ":iframe input#other_amount:not(:checked)",
         },
+        {
+            content: "Click on the custom amount radio button",
+            trigger: ":iframe input#other_amount",
+            run: "click",
+        },
+        {
+            content: "Verify that other_amount is checked",
+            trigger: ":iframe input#other_amount:checked",
+        },
+        {
+            content: "Submit the donation form",
+            trigger: ":iframe button[name='o_payment_submit_button']",
+            run: "click",
+        },
+        {
+            content: "Check warning message is display",
+            trigger: ":iframe .o_donation_payment_form:has(p#warningMessageId)",
+        },
+        {
+            content: "Enter an amount less than the minimum value",
+            trigger: ":iframe input#other_amount_value",
+            run: "edit 1",
+        },
+        {
+            content: "Verify the display of a minimum value warning message",
+            trigger: ":iframe p#warningMinMessageId:contains('The minimum donation amount is $5')",
```

Comment:
Why didn't you take this remark into consideration ? https://github.com/odoo/odoo/pull/182490/files#r1832114860